src/tasks/dump.ts

```diff
@@ -460,15 +460,46 @@ async function createOrders(
   receiver: Receiver,
   domainSeparator: TypedDataDomain,
   validTo: number,
+  maxFeePercent: number,
   api: Api,
 ) {
   for (const inst of instructions) {
+    const sellToken = inst.token.address;
+    const buyToken = isNativeToken(toToken) ? BUY_ETH_ADDRESS : toToken.address;
+
+    // Re-quote for up-to-date fee (in case approval took long)
+    const updatedQuote = await api.getQuote({
```

Comment:
@nlordell is this one these uncaught promises you were mentioning that could crash the script?

Reply:
The uncaught promise would be one without an `await`.